tkg/client/delete_region.go

```diff
@@ -152,6 +152,15 @@ func (c *TkgClient) DeleteRegion(options DeleteRegionOptions) error { //nolint:f
 
 		// If clusterclass feature flag is enabled then deploy management components
 		if config.IsFeatureActivated(config.FeatureFlagPackageBasedLCM) {
+			// Read config variable from management cluster and set it into cleanup cluster
+			configValues, err := c.getUserConfigVariableValueMapForUpgrade(regionContext.SourceFilePath, regionContext.ContextName)
```

Comment:
Maybe we can refactor the method name to `getUserConfigVariableValueMapFromManagementCluster`. `ForUpgrade` is a bit confusing.

Reply:
Thank you @DanielXiao for reviewing, yes, `ForUpgrade` is indeed make no sense now, considering the mgmt cluster context is just a parameter, maybe using `getUserConfigVariableValueMapFromSecret` or `getUserConfigVariableValueMapFromCluster` comparing to the other one that's getting them from config file, and I see similar naming methods used in clusterclient.go

Reply:
Yes, `getUserConfigVariableValueMapFromSecret` looks good

Reply:
Addressed